src/core/bpf/restrict_ifaces/restrict-ifaces.bpf.c

```diff
@@ -0,0 +1,52 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+/* <linux/bpf.h> must precede <bpf/bpf_helpers.h> due to integer types
+ * in bpf helpers signatures.
+ */
+#include <linux/bpf.h>
+#include <bpf/bpf_helpers.h>
+
+const volatile __u8 is_allow_list = 0;
```

Comment:
how does this actually work? i.e. how does the consuming code get access to this, and in particular write access?

Reply:
The generated skeleton includes a structure with those variables, they are filled by the user space part (`obj->rodata->is_allow_list = is_allow_list;` in restrict-ifaces.c) before loading the BPF object. When the object is loaded, libbpf creates an eBPF map that backes up those variables, libbpf copies the value of those to the map and freezes it to avoid further modifications in the case the variable is const.